datasets/fr/illegal_financial_services/crawler.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from typing import Dict
+from rigour.mime.types import CSV
+from normality import slugify
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(row.get("nom"))
+    entity.add("name", row.pop("nom"))
```

Comment:
we can probably do a little regex so that if it looks roughly like a url (with/without http*) we add it as website, and if it looks roughly like an email address we add it as email